docs/MIGRATING_TO_1_0.md

```diff
@@ -0,0 +1,163 @@
+# Migrating from 0.3 to 1.0
+
+## Split in two modules
+
+The core of immutad●t has been rewritten without [lodash](https://lodash.com/), and immutad●t now has two modules [immutadot](https://www.npmjs.com/package/immutadot) and [immutadot-lodash](https://www.npmjs.com/package/immutadot-lodash).
+
+### immutadot module
+
+immutadot module now contains exclusively functions based on [ES2015+ standard library](https://mdn.io/JavaScript/Reference).
```

Comment:
and a few common functions like `toggle`?

Reply:
I would say based on "based on ES2015+ language and standard library".
`toggle` is just `!` which is part of the language.
What do you think ?

Reply:
standard library is a bit vague, no ?